java/ql/test/utils/model-generator/typebasedflow/p/Stream.java

```diff
@@ -0,0 +1,249 @@
+        return null;
+    }
+
+    // Model generator adds a couple of extra models, which can't be
+    // dismissed based on the type information.
+    // MaD=p;Stream;true;iterate;(Object,UnaryOperator);;Argument[0];Argument[1].Parameter[0];value;generated
+    // MaD=p;Stream;true;iterate;(Object,UnaryOperator);;Argument[0];ReturnValue.Element;value;generated
+    // MaD=p;Stream;true;iterate;(Object,UnaryOperator);;Argument[1].ReturnValue;Argument[1].Parameter[0];value;generated
+    // MaD=p;Stream;true;iterate;(Object,UnaryOperator);;Argument[1].ReturnValue;ReturnValue.Element;value;generated
+    public static <T> Stream<T> iterate(T seed, UnaryOperator<T> f) {
+        return null;
+    }
+
+    // MaD=p;Stream;true;limit;(long);;Argument[-1].Element;ReturnValue.Element;value;generated
+    public Stream<T> limit(long maxSize) {
+        return null;
+    }
+
+    // MaD=p;Stream;true;map;(Function);;Argument[-1].Element;Argument[0].Parameter[0];value;generated
+    // MaD=p;Stream;true;map;(Function);;Argument[0].ReturnValue;ReturnValue.Element;value;generated
+    public <R> Stream<R> map(Function<? super T, ? extends R> mapper) {
+        return null;
+    }
+
+    // MaD=p;Stream;true;mapToDouble;(ToDoubleFunction);;Argument[-1].Element;Argument[0].Parameter[0];value;generated
+    public DoubleStream mapToDouble(ToDoubleFunction<? super T> mapper) {
+        return null;
+    }
+
+    // MaD=p;Stream;true;mapToInt;(ToIntFunction);;Argument[-1].Element;Argument[0].Parameter[0];value;generated
+    public IntStream mapToInt(ToIntFunction<? super T> mapper) {
+        return null;
+    }
+
+    // MaD=p;Stream;true;mapToLong;(ToLongFunction);;Argument[-1].Element;Argument[0].Parameter[0];value;generated
+    public LongStream mapToLong(ToLongFunction<? super T> mapper) {
+        return null;
+    }
+
+    // MaD=p;Stream;true;max;(Comparator);;Argument[-1].Element;Argument[0].Parameter[0];value;generated
+    // MaD=p;Stream;true;max;(Comparator);;Argument[-1].Element;Argument[0].Parameter[1];value;generated
+    // MaD=p;Stream;true;max;(Comparator);;Argument[-1].Element;ReturnValue.SyntheticField[ArgType0];value;generated
+    public Optional<T> max(Comparator<? super T> comparator) {
+        return null;
+    }
+
+    // MaD=p;Stream;true;min;(Comparator);;Argument[-1].Element;Argument[0].Parameter[0];value;generated
+    // MaD=p;Stream;true;min;(Comparator);;Argument[-1].Element;Argument[0].Parameter[1];value;generated
+    // MaD=p;Stream;true;min;(Comparator);;Argument[-1].Element;ReturnValue.SyntheticField[ArgType0];value;generated
+    public Optional<T> min(Comparator<? super T> comparator) {
+        return null;
+    }
+
+    // MaD=p;Stream;true;noneMatch;(Predicate);;Argument[-1].Element;Argument[0].Parameter[0];value;generated
+    public boolean noneMatch(Predicate<? super T> predicate) {
+        return false;
+    }
+
+    // Issue with model generator. ... is not supported.
```

Comment:
That's fine, but no reason to comment the test case.

Reply:
I will comment it in.